### PR TITLE
ACP-7: Show customer details

### PR DESCRIPTION
**Pull request by jonholmberg**
[Ticket](https://jira.e-conomic.net/browse/ACP-7)

### CODE REVIEW

File: Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomerDetailsActivity.kt [hunk 1]
```diff
@@ -13,21 +17,47 @@ class CustomerDetailsActivity : CustomersBaseActivity() {
     override val layoutId = R.layout.activity_customerdetails
 
     val toolbar by lazy { supportActionBar }
+    val nameView by lazy { edit_text_customer_name }
+    val telephoneAndFaxNumberView by lazy { edit_text_customer_telephoneAndFaxNumber }
+    val emailView by lazy { edit_text_customer_email }
+    val addressView by lazy { edit_text_customer_address }
+    val cityView by lazy { edit_text_customer_city }
+    val zipView by lazy { edit_text_customer_zip }
+    val countryView by lazy { edit_text_customer_country }
```
Comment (ned1988) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomerDetailsActivity.kt]: Cool I like this way, its looks nice.

File: Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomerDetailsActivity.kt [hunk 2]
```diff
@@ -13,21 +17,47 @@ class CustomerDetailsActivity : CustomersBaseActivity() {
...
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        toolbar?.setDisplayHomeAsUpEnabled(true)
-    }
+    private lateinit var customer: Customer
 
     override fun updateUIFromStore() {
-        // TODO: Implement in ACP-7 / ACP-10
+        runOnUiThread {
```
Comment (ned1988) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomerDetailsActivity.kt]: Do we need `runOnUiThread ` here? or we call this method also from background thread?

Comment (jonholmberg) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomerDetailsActivity.kt]: We don't do it as of right now, but it was just to be sure that you could never update the UI from a background thread. Should I remove it?

Comment (ned1988) [inline Customers/app/src/main/java/com/e_conomic/customers/ui/activities/CustomerDetailsActivity.kt]: Hard to say, if we can't run this code in background I think we don't need this line.

File: Customers/app/src/main/res/layout/activity_customerdetails.xml [hunk 3]
```diff
@@ -1,8 +1,213 @@
...
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <android.support.v7.widget.CardView
```
Comment (jonholmberg) [inline Customers/app/src/main/res/layout/activity_customerdetails.xml]: I'm not entirely sure how you thought this to be done as the card views have quite different layouts inside. But I've made some changes which we can discuss when you get back. For now I think I will merge this so I can continue :)

**Pull request merged**